### ISSUE TITLE
[BUG]: Thank you message doesn't time out

### ISSUE THREAD

**Issue by michaella23** [TheTechCommute/TheTechCommute#59]
**Describe the bug**
When someone signs up for email updates, they get a 'Thank you' message. 
There are a few issues:
- the color contrast is not accessible (issue #58)
- the display is a bit jumpy due to the styling
- the 'Thank you' message does not clear unless you refresh the page

**To Reproduce**
1. Go to https://thetc.co/
2. Click on 'Subscribe' (after entering an email address)
3. See 'Thank you message appear'

**Expected behavior**
I expected the container to stay the same size, even if the text changed when I clicked subscribe. But after it changed I expected it to return to its original size.

**Screenshots**

https://github.com/user-attachments/assets/0921c790-de47-45dc-8c7e-b6247c7d713e

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser: Chrome

**Additional context**
A separate issue was created for the color contrast.

**Proposed solution**
Have an empty `p` element to hold that space and fill it with the 'Thank You' message. A timeout for the message may not be necessary if the display isn't changing so dramatically.

**Comment by michaella23** [comment]
Also, I had already Subscribed before this screen recording, so when I clicked the button the first time nothing happened. The second click triggered the 'Thank you' message.

**Comment by imkrishnasarathi** [comment]
I could work on this if assigned!

**Comment by michaella23** [comment]
go for it @imkrishnasarathi ! Thanks!
